markup/house-style.md

```diff
@@ -30,9 +30,11 @@ As such it seems beneficial to choose HTML 5 over XHTML.
 
 ### Validation
 
-We do what is reasonable to ensure our documents [validate](https://validator.w3.org/).
+Do what is reasonable to ensure your documents [validate](https://validator.w3.org/). 
 
-Invalid markup results in authors relying on all clients to do what they intended, not what they instructed.  Clients regularly disagree on how to interpret out instructions, yet alone our intentions, so to promote consistency of execution we must do what we can to ensure our HTML is valid.
+Invalid markup results in authors relying on all clients to do what they intended, not what they instructed.  Clients regularly disagree on how to interpret out instructions, yet alone our intentions, so to promote consistency of execution you must do what you can to ensure your HTML is valid.
```

Comment:
my typo not yours but `out instructions`

Reply:
good catch, sorted now 👍